exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyFormatPlugin.java

```diff
@@ -173,7 +173,7 @@ CloseableRecordBatch getReaderBatch(FragmentContext context, EasySubScan scan) t
     }
 
     for(FileWork work : scan.getWorkUnits()){
-      readers.add(getRecordReader(context, dfs, work, scan.getColumns()));
+      readers.add(getRecordReader(context, dfs, work, scan.getColumns(), scan.getUserName()));
```

Comment:
Is it possible to pass the proxyUgi object instead of creating the object for every reader?

Currently ugi objects are not cached.